README.md

```diff
@@ -1,59 +1,67 @@
+    2. Copy the **Application (client) ID** value
+    3. In Mattermost, go to **System Console -> Plugins -> Skype for Business**, and enter the **Application ID** you copied on the previous step
+    4. Choose ``Online`` as the **Skype for Business Product Type**.
+    5. Click **Save**
 6. Enable the plugin
-    1. Go to System Console -> Plugins -> Management and click "Enable" underneath the Skype For Business plugin
+    1. Go to **System Console -> Plugins -> Management** and click **Enable** underneath the Skype for Business plugin
 7. Try it out
-    1. Click the Skype for Business icon in the channel header
-    2. A popup will be opened to sign in on the microsoftonline.com
-    3. Once you signed in, the popup will be automatically closed
-    4. New post will be created with a link to the newly created meeting
-    5. By clicking the "Join meeting" button, you will be redirected to a meet.lync.com page, which in turn will open a Skype for Business client installed on your computer / smartphone to join the meeting
+    1. Go to a Mattermost channel and click the Skype for Business icon in the channel header
+    2. A dialog will open asking you to sign in to microsoftonline.com
+    3. Once signed in, the dialog will close and a new message posted to the channel, with a link to the newly created meeting
+    4. By clicking the "Join meeting" button, you will be redirected to a meet.<YOUR DOMAIN> page, which in turn opens a Skype for Business client installed on your computer or smartphone to join the meeting
 
 ### Skype for Business Server
 
 1. Install the plugin
-    1. Clone the project from GitHub onto your local computer and build it
-    2. In Mattermost, go the System Console -> Plugins -> Management
+    1. Download the latest version of the plugin from the GitHub releases page
+    2. In Mattermost, go to **System Console -> Plugins -> Management**
     3. Upload the plugin
-2. Create a bot account in your domain.
-3. Fill the "Bot username" and "Bot password" fields with the bot's account credentials.
-    1. Set "Is Server version?" to true
-    2. Fill the "Domain" field with the domain where your Skype for Business Server is running, e.g. contoso.com.
-4. Enable the plugin
-    1. Go to System Console -> Plugins -> Management and click "Enable" underneath the Skype For Business plugin
-5. Try it out
-    1. Click the Skype for Business icon in the channel header
-    2. New post will be created with a link to the newly created meeting
-    3. By clicking the "Join meeting" button, you will be redirected to a meet.<YOUR DOMAIN> page, which in turn will open a Skype for Business client installed on your computer / smartphone to join the meeting
+2. Configure the plugin
+    1. Go to **System Console -> Plugins -> Skype for Business**
+    2. Choose ``Server`` as the **Skype for Business Product Type**
+    3. Enter the **Server Domain** of your Skype for Business server instance. For example, ``contoso.com.``
```

Comment:
As we agreed to save the credentials in the plugin settings, what about adding new lines with the "Bot username" and the "Bot password" fields? Maybe something like this:

4. Enter the ***Bot username*** of your bot user created in the Skype for Business domain. For example, ``bot@contoso.com``
5. Enter the ***Bot password*** of your bot user created in the Skype for Business domain

?

Reply:
Indeed 👍